### PR TITLE
Ncnews 2

### PR DESCRIPTION
**Pull request by Ha-Mundo**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+const express = require("express");
+const { psqlErrors, customErrors, handle500s } = require("./errors/index");
+const { getTopics } = require("./controllers/topics.controllers.js");
+const app = express();
+app.use(express.json());
```
Comment (itskatherine) [inline app.js:5]: This isn't being used yet, it's only needed when a request has a body

File: errors/index.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    res.status(err.status).send({ msg: err.msg });
+  } else next(err);
+};
+
+exports.psqlErrors = (err, req, res, next) => {
+  if (err.code === "22P02" || err.code === "23502") {
+    res.status(400).send({ msg: "bad request" });
+  } else next(err);
+};
+
```
Comment (itskatherine) [inline errors/index.js:12]: Really great error handling middleware functions, awesome work
However... you're not handling any errors in this ticket and you won't have come across any of the errors these functions are catching yet... so these are unnecessary until you start error handling.

**Comment by itskatherine** [comment]
Have a look at comments I've left, feel free to merge, but in the future watch out for code brought in before it is needed or functional.

File: app.js (hunk 1 above)
Comment (Ha-Mundo) [reply app.js:5]: Understood.

File: errors/index.js (hunk 2 above)
Comment (Ha-Mundo) [reply errors/index.js:12]: OK

**Pull request merged**